src/MusicPlayer.cpp

```diff
@@ -14,4 +15,17 @@ void MusicPlayer::Toggle() {
     else {
         music.play();
     }
-}
\ No newline at end of file
+}
+void MusicPlayer::Play() {
+    if (music.getStatus() == sf::SoundSource::Status::Paused) {
```

Comment:
it would be better to change it to music.getStatus() != sf::SoundSource::Status::Playing? Will be more safe (in case the music was actually stopped and not paused, we still want Play to work).

Also, if you check the SFML reference (https://www.sfml-dev.org/documentation/2.5.1/classsf_1_1SoundStream.php#a932ff181e661503cad288b4bb6fe45ca) you can see, that calling play while the music is already playing, simply restarts it. So you don't even have to do any checks and can play the music just like that (unless you specifically don't want the user to allow restarting the music like that, then you need it)